### DISCUSSION TITLE
how to make work on laravel 6

### DISCUSSION THREAD

**Discussion by sarmenb** [spatie/ray#496] [Q&A]
does ray work on laravel 6? I'm working with a laravel 6 project and cant update it to the newer version of laravel so im stuck with version 6 for now. is there a way to make ray work with this version? it would save my life 10 times over.

**Comment by sinnbeck** [comment]
Try this package: https://github.com/xndbogdan/laravel-ray-legacy

Seems to have been made for legacy
